tests/UI/campaigns/functional/BO/00_login/01_passwordReminder.js

```diff
@@ -0,0 +1,273 @@
+
+      const alertSuccessMessage = await emailPage.setupSmtpParameters(
+        page,
+        smtpServer,
+        DefaultCustomer.email,
+        DefaultCustomer.password,
+        smtpPort,
+      );
+
+      await expect(alertSuccessMessage).to.contains(emailPage.successfulUpdateMessage);
+    });
+
+    it('should check successful message after sending test email', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sendTestEmail', baseContext);
+
+      const textResult = await emailPage.sendTestEmail(page, global.BO.EMAIL);
+      await expect(textResult).to.contains(emailPage.sendTestEmailSuccessfulMessage);
+    });
+
+    it('should check if test mail is received', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkMailBox', baseContext);
+
+      await expect(newMail.subject).to.contains(testMailSubject);
+    });
+
+    it('should logout from BO', async function () {
+      await loginCommon.logoutBO(this, page);
+    });
+  });
+
+  describe('Go to BO and create a new employee', async () => {
+    it('should login in BO', async function () {
+      await loginCommon.loginBO(this, page);
+    });
+
+    it('should go to "Advanced parameters>Team" page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAdvancedParamsPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.advancedParametersLink,
+        dashboardPage.teamLink,
+      );
+
+      await employeesPage.closeSfToolBar(page);
+
+      const pageTitle = await employeesPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(employeesPage.pageTitle);
+    });
+
+    it('should reset all filters and get number of employees', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
+      numberOfEmployees = await employeesPage.resetAndGetNumberOfLines(page);
+      await expect(numberOfEmployees).to.be.above(0);
+    });
+
+    it('should go to add new employee page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToNewEmployeePage', baseContext);
+
+      await employeesPage.goToAddNewEmployeePage(page);
+      const pageTitle = await addEmployeePage.getPageTitle(page);
+      await expect(pageTitle).to.contains(addEmployeePage.pageTitleCreate);
+    });
+
+    it('should create employee and check result', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'createEmployee', baseContext);
+
+      const textResult = await addEmployeePage.createEditEmployee(page, createEmployeeData);
+      await expect(textResult).to.equal(employeesPage.successfulCreationMessage);
+
+      const numberOfEmployeesAfterCreation = await employeesPage.getNumberOfElementInGrid(page);
+      await expect(numberOfEmployeesAfterCreation).to.be.equal(numberOfEmployees + 1);
```

Comment:
```suggestion
      await expect(textResult).to.equal(employeesPage.successfulCreationMessage);
```

No need for this check here (to part of the test)